definitions/npm/is-number_v1.x.x/test_is-number_v1.x.x.js

```diff
@@ -0,0 +1,10 @@
+import isNumber from 'is-number';
+
+const a: boolean = isNumber(4);
```

Comment:
Could you please wrap tests in a few [`describe/it` blocks](https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#use-describe-and-it-blocks-to-limit-scope)?

Reply:
Fixed